### PR TITLE
Enable milestone plugin for the Kubernetes org.

### LINKED ISSUE #7096: Milestone maintainer is unable to set milestone via milestone command
Per #6982, Milestone maintainers should be able to set a milestone. However, I tested it and found it does not work.

I tested both `/milestone v1.10` and `/milestone v1.11` it in PR: https://github.com/kubernetes/kubernetes/pull/59720 and didn't get a response.

I am a milestone maintainer(xref: https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers/members?utf8=%E2%9C%93&query=m1093782566) and can approve a milestone via command `/status approved-for-milestone`, but there is no response when I type a `/milestone` command.

Am I missing something?

/cc @spiffxp @grodrigues3 @BenTheElder @kargakis @cjwagner @stevekuznetsov

/kind bug
/area prow

### PR DESCRIPTION
**Pull request by cjwagner**
fixes #7096 
/hold

### CODE REVIEW

**Comment by cjwagner** [comment]
Oops, I didn't see #7097 🤦‍♂️ 

Hook will need an image bump like the one I did in this PR so lets keep this PR and close the other.

**Comment by BenTheElder** [comment]
Why is hook being bumped here?

**Comment by cjwagner** [comment]
Added the godoc comment fix that @m1093782566 had made. Please restamp this PR. 
(The `lgtm` label removal on commit push is working now 🎉)

File: prow/cluster/hook_deployment.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
       - name: hook
-        image: gcr.io/k8s-prow/hook:v20180302-0064877cc
+        image: gcr.io/k8s-prow/hook:v20180302-5519b614a
```
Comment (BenTheElder) [inline prow/cluster/hook_deployment.yaml]: why did this change? 0064877cc -> 0064877cc7181a16a6d753d111eb1381bfdaa148, which is already past the milestone updates.

File: prow/hook/plugins.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ import (
 	_ "k8s.io/test-infra/prow/plugins/label"
 	_ "k8s.io/test-infra/prow/plugins/lgtm"
 	_ "k8s.io/test-infra/prow/plugins/lifecycle"
+	_ "k8s.io/test-infra/prow/plugins/milestone"
```
Comment (cjwagner) [inline prow/hook/plugins.go]: > Why is hook being bumped here?

Hook is being bumped to compile the milestone plugin into the binary.

Comment (BenTheElder) [inline prow/hook/plugins.go]: Oh...

**Comment by BenTheElder** [comment]
Let's roll it out then :-)

**Comment by m1093782566** [comment]
Sorry, but I am still unable to set a milestone via `milestone` command. I tested it in https://github.com/kubernetes/kubernetes/pull/59720, please check it out.

Thanks!

**Pull request merged**